aeron-client/src/main/java/io/aeron/Publication.java

```diff
@@ -103,7 +103,7 @@ protected Publication(
         final long registrationId,
         final int maxMessageLength)
     {
-        final UnsafeBuffer logMetaDataBuffer = logBuffers.metaDataBuffer();
```

Comment:
This change would involve repeated field access rather than reuse from a register.

Reply:
reverted.